tests/wsl/prepare_wsl.pm

```diff
@@ -24,6 +24,9 @@ sub run {
     my ($self) = @_;
 
     $self->open_powershell_as_admin;
+    # WSL must be in the latest version, otherwise the `--no-distribution`
+    # option is not available.
+    $self->run_in_powershell(cmd => 'wsl.exe --update', timeout => 300);
```

Comment:
This actually only works on recent Win 11 versions, that's why the `if get_var('HDD_1') =~ /24H2/;` conditional is there.